.babelrc

```diff
@@ -1,13 +1,14 @@
 {
   "presets": [
     [
-      "env",
```

Comment:
Why?

Reply:
Just to be consistent!

Reply:
With what? Use just used shorthand in this file `transform-object-rest-spread`.

Reply:
Babel 7 is coming and will remove shorthand, just psychologic preparation.